tests/expectedoutputs/TestWMSTimeHeightProfiles/test_WMSGetCapabilities_TimeHeightProfiles.xml

```diff
@@ -0,0 +1,128 @@
+        </Exception>
+        <UserDefinedSymbolization SupportSLD="1" />
+<Layer>
+<Title>WMS of  CHM15k Nimbus</Title>
+<CRS>EPSG:3411</CRS>
+<CRS>EPSG:3412</CRS>
+<CRS>EPSG:3575</CRS>
+<CRS>EPSG:3857</CRS>
+<CRS>EPSG:4258</CRS>
+<CRS>EPSG:4326</CRS>
+<CRS>CRS:84</CRS>
+<CRS>EPSG:25831</CRS>
+<CRS>EPSG:25832</CRS>
+<CRS>EPSG:28992</CRS>
+<CRS>EPSG:7399</CRS>
+<CRS>EPSG:50001</CRS>
+<CRS>EPSG:54030</CRS>
+<CRS>EPSG:32661</CRS>
+<CRS>EPSG:40000</CRS>
+<CRS>EPSG:900913</CRS>
+<CRS>EPSG:3067</CRS>
+<BoundingBox CRS="EPSG:3411" minx="3181985.726484" miny="-2936325.838930" maxx="3322855.942587" maxy="-2799483.358645" />
+<BoundingBox CRS="EPSG:3412" minx="1867723.902828" miny="34495333.339115" maxx="2540087.501323" maxy="35511264.460837" />
+<BoundingBox CRS="EPSG:3575" minx="-513615.657195" miny="-4250014.213132" maxx="-428706.448090" maxy="-4137435.259796" />
+<BoundingBox CRS="EPSG:3857" minx="344625.100433" miny="6610924.247303" maxx="455944.591226" maxy="6789526.168440" />
+<BoundingBox CRS="EPSG:4258" minx="50.941341" miny="3.095820" maxx="51.941341" maxy="4.095820" />
+<BoundingBox CRS="EPSG:4326" minx="50.941341" miny="3.095820" maxx="51.941341" maxy="4.095820" />
+<BoundingBox CRS="CRS:84" minx="50.941341" miny="3.095820" maxx="51.941341" maxy="4.095820" />
+<BoundingBox CRS="EPSG:25831" minx="506586.685443" miny="5643306.234342" maxx="576989.198618" maxy="5755081.271384" />
+<BoundingBox CRS="EPSG:25832" minx="85335.519937" miny="5654761.926157" maxx="162984.338772" maxy="5770997.834634" />
+<BoundingBox CRS="EPSG:28992" minx="-6037.898649" miny="328758.635711" maxx="66205.391192" maxy="441693.548928" />
+<BoundingBox CRS="EPSG:7399" minx="229341.630247" miny="5971691.317382" maxx="307650.224519" maxy="6075231.074174" />
+<BoundingBox CRS="EPSG:50001" minx="-2000000.000000" miny="-2000000.000000" maxx="10000000.000000" maxy="8500000.000000" />
+<BoundingBox CRS="EPSG:54030" minx="250267.819665" miny="5424087.672457" maxx="333585.083442" maxy="5526906.416450" />
+<BoundingBox CRS="EPSG:32661" minx="2236636.957682" miny="-2499207.579300" maxx="2321824.106000" maxy="-2370491.121972" />
+<BoundingBox CRS="EPSG:40000" minx="238065.349781" miny="-4526365.773944" maxx="323766.706237" maxy="-4396872.356109" />
+<BoundingBox CRS="EPSG:900913" minx="344625.100433" miny="6610924.247303" maxx="455944.591226" maxy="6789526.168440" />
+<BoundingBox CRS="EPSG:3067" minx="-1168100.884114" miny="5897608.814895" maxx="-1063341.469832" maxy="6029339.633032" />
+<Layer queryable="1" opaque="1" cascaded="0">
+<Name>beta_raw</Name>
+<Title>normalized range corrected signal
+((P_raw / lp) - b) / (cs * o(r) * p_calc) * r * r
+with P_raw = sum(P_raw_hr) * range_gate_hr / range_gate (beta_raw)</Title>
+<EX_GeographicBoundingBox>
+  <westBoundLongitude>3.095820</westBoundLongitude>
+  <eastBoundLongitude>4.095820</eastBoundLongitude>
+  <southBoundLatitude>50.941341</southBoundLatitude>
+  <northBoundLatitude>51.941341</northBoundLatitude>
```

Comment:
This is a measurement for a single point, correct?
Isn't it a bit strange to give this an extent so that a viewer (Geoweb?) will handle it correctly? Or did I misunderstand?

Reply:
If you zoom to layer, you see the station in the middle of the map, with margins of .5 degree around it.

WMS clients expect a certain boundingbox, in this case it is 1 degree big. I do not see any problem with that.

Reply:
This still feels strange to me. Shouldn't the GetCapabilities describe the dataset?

Anyway, you have more experience with this.

Reply:
Certain clients will use the width of a boundingbox by substracting `east` - `west`. (Which is then `zero`). Some clients (also adaguc-viewer and GeoWeb) will crash because of a division by zero issue. Also they will zoom in to a infinitely small area, causing issues. That is why the extent of a WMS layer needs to have a certain area.

Reply:
Okay, I'll accept it. To help all the broken clients.

Might be worth fixing that in Geoweb, though...